src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -37,6 +39,9 @@ import {
   extractLocation,
   prepareForSchedule,
 } from "./utils";
+
+import { createCalendar } from "utils/calendar";
+
 import { isEventWithinDate } from "utils/event";
 import { WithVenueId } from "utils/id";
```

Comment:
All of these `utils/*` imports (not the `./utils` import) should be moved up after `settings` import, before the `hook/*` imports, with a single line of whitespace before/after the `utils/*` block of imports (and no spaces in between them)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1423/commits/ec68e834fab42e9549a285d7a454e4c29e1f24e4